packages/drawer/src/views/modern/Drawer.tsx

```diff
@@ -210,7 +210,9 @@ export default function Drawer({
           : open;
 
       toggleDrawer(nextOpen, event.velocityX);
-      runOnJS(onGestureEnd)();
+    },
+    onFinish: () => {
+      runOnJS(onGestureFinish)();
```

Comment:
thanks for the PR. Maybe `onGestureFinish` does not need to exists and we just call `endInteraction` here?

Reply:
Thanks @vonovak, I think it is good as is since the `onGestureFinish` JS callback matches `onGestureStart`. Both callbacks existed before this PR, and I only renamed `onGestureEnd` to `onGestureFinish`.

Let me know we should remove `onGestureFinish` and only call `endInteraction` here and I will make the change.